docs/user-guide/persistent-volumes/index.md

```diff
@@ -250,6 +256,10 @@ spec:
 
 Claims use the same conventions as volumes when requesting storage with specific access modes.
 
+### Volume Modes
+
+Claims use the same convention as volumes to indicates the consumption of the volume as either a filesystem or block device.
+
```

Comment:
Please explain available values for volumeMode too. unspecified/Filesystem/Block
Also we need to explain about what kind of volumeMode combinations could be bound.